webapp/src/main/webapp/vue-apps/external-visio/main.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2024 eXo Platform SAS.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+import './initComponents.js';
+
+
+if (extensionRegistry) {
+  const components = extensionRegistry.loadComponents('externalVisio');
+  if (components && components.length > 0) {
+    components.forEach(cmp => {
+      Vue.component(cmp.componentName, cmp.componentOptions);
+    });
+  }
+}
+
+Vue.use(Vuetify);
+const vuetify = new Vuetify(eXo.env.portal.vuetifyPreset);
+
+const appId = 'ExternalVideoAdmin';
+
+//getting language of the PLF
+const lang = eXo?.env.portal.language || 'en';
+const url = `${eXo.env.portal.context}/${eXo.env.portal.rest}/i18n/bundle/locale.portlet..externalVideo.ExternalVideo-${lang}.json`;
+
+
+extensionRegistry.registerComponent('manageExternalVisio', 'manage-external-visio', {
```

Comment:
Here you create an extension point in Visio Admin Portlet, naming it manageExternalVisio. 
So you make it 'specific' to external visio as it is an extension point. 
Perhaps in the future, we will have another visio connector, named 'whateverVisioConnector', which need to use this extension point. 
So please rename this extension point to something not specific, for example
```suggestion
extensionRegistry.registerComponent('additional-visio-configration', 'manage-external-visio', {
```